grapher/stackedCharts/StackedAreaChart.tsx

```diff
@@ -308,6 +228,75 @@ export class StackedAreaChart
         )
     }
 
+    @action.bound private onCursorMove(
+        ev: React.MouseEvent<SVGGElement> | React.TouchEvent<SVGElement>
+    ): void {
+        if (!this.base.current) return
+        const { dualAxis, series } = this
+
+        const mouse = getRelativeMouse(this.base.current, ev.nativeEvent)
+
+        const boxPadding = isMobile() ? 44 : 25
```

Comment:
These constants are the same between LineChart and StackedArea, just wondering if they should be shared variables, since it's unlikely you want to change one without changing the other (they are very similar in the hover behaviour).

Reply:
I agree in general, but wouldn't know where to put them? They are too specific for `Util.tsx` I think?